packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -69,89 +93,217 @@ export class LitDevTutorial extends LitElement {
   @state()
   private _preloadedHtml?: {idx: number; promise: Promise<string>};
 
+  /**
+   * Tutorial manifest json file
+   */
+  @state()
+  private _manifest: TutorialManifest = {steps: [{title: ''}]};
+
   /**
    * Whether the tutorial is currently in its "solved" state.
    */
   private _solved = false;
 
+  private get _projectLocation() {
+    // TODO(e111077): delete this once we go public with tutorials catalog
+    if (window.location.pathname === '/tutorial/') {
+      return 'intro-to-lit';
+    }
+    return window.location.pathname.replace('/tutorials/', '');
+  }
+
+  /**
+   * Playground project element generated from this.project
```

Comment:
```suggestion
   * Playground project element discovered using the id from this.project
```

("generated" kind of made it sound like it was being created here)